octoprint_displayprogress/static/css/displayprogress.css

```diff
@@ -0,0 +1,47 @@
+/* Thanks https://websemantics.uk/tools/image-to-data-uri-converter/ for the quick png to dataurl code! */
+
+div#settings_plugin_displayprogress #preview {
+  display: inline-block;
+  height: 40px;
+  margin-left: 20px;
+}
+
+div#settings_plugin_displayprogress label {
+  font-size: larger;
+  font-weight: bolder;
+}
+
+div#settings_plugin_displayprogress div.controls select{
+  width: 250px;
+  font-size: large;
+  font-weight: bolder;
+  padding-top: 0;
+}
+
+#preview {
```

Comment:
Should be prefixed with `settings_plugin_displayprogress`, otherwise we risk conflicts with core code or other plugins. Same goes for the individual symbol classes further below.